js/zb.js

```diff
@@ -4245,6 +4246,63 @@ module.exports = class zb extends Exchange {
         return rates;
     }
 
+    async borrowMargin (code, amount, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
```

Comment:
Let's add `symbol = market['symbol']` in such cases, cause the user might have passed a marketId instead of a symbol, and this way we fix it by rewriting it from the proper field of the proper market – that helps filtering by symbol afterwards.

```suggestion
            market = this.market (symbol);
            symbol = market['symbol']
```